packages/contracts-periphery/contracts/foundry-tests/citizen-house/BadgeAdmin.t.sol

```diff
@@ -0,0 +1,692 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity 0.8.15;
+
+import { BadgeAdmin } from "../../universal/citizen-house/BadgeAdmin.sol";
+import { Badge } from "../../universal/citizen-house/Badge.sol";
+import { Test, stdError } from "forge-std/Test.sol";
+
+function mkadr(string memory name) pure returns (address) {
+    address adr = address(uint160(uint256(keccak256(abi.encodePacked(name)))));
+    return adr;
+}
+
+contract TBadge is Badge {
+    constructor() Badge("example", "ex", "example.com") {}
+}
+
+contract TBadgeAdmin is BadgeAdmin {
+    TBadge internal badge;
+
+    address[] opAdrs = [mkadr("op")];
+    address[] adrs = [0xa8B3478A436e8B909B5E9636090F2B15f9B311e7];
+
+    function setUp() public {
+        badge = new TBadge();
+    }
+
+    constructor() BadgeAdmin(address(badge), 100, 100, 100, opAdrs) {}
+}
+
+contract BadgeAdminTest is Test {
+    TBadgeAdmin internal badgeAdmin;
+    TBadge internal badge;
+
+    address opAdr = mkadr("op"); //[0xa8B3478A436e8B909B5E9636090F2B15f9B311e7];
```

Comment:
Dead code? The forge-std helper will label the address for you so it'll appear nicely in traces